zkevm-circuits/src/super_circuit.rs

```diff
@@ -260,6 +260,12 @@ impl<F: Field, const MAX_TXS: usize, const MAX_CALLDATA: usize, const MAX_RWS: u
             block.circuits_params.max_rws,
             Value::known(block.randomness),
         )?;
+        config.tx_table.load(
```

Comment:
The TxTable is already loaded when calling `self.tx_circuit.assign`:
https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/3b06585106417a398a0af37422dc1fcadf5a614c/zkevm-circuits/src/tx_circuit.rs#L169

So there should be no need to load it again.

Reply:
ok.. understand. so i think later we should unify the "tx table assignment in evm circuit" and "tx circuit assignment"..

Reply:
In general the idea is that each table has a circuit that is responsible to verify its correctness, and the table is contained within the circuit.  The assign function of such circuit is also responsible of assigning/loading the table.
So:
- keccak circuit assigns the keccak table
- tx circuit assigns the tx table
- copy circuit assigns the copy table
- mpt circuit assigns the mpt table (not yet in main)
- bytecode circuit assigns the bytecode table
- exp circuit assigns the exp table
- state circuit assigns the rw table (this is currently not being done)
- block circuit assigns the block table (we don't have a block circuit yet)

Nevertheless we also want unit tests with single circuits, and for those we also need the tables loaded, for which we have the `Table.load` functions.